.github/workflows/wasm.yml

```diff
@@ -115,13 +115,13 @@ jobs:
       - name: Upload artifact
         uses: actions/upload-artifact@v3
         with:
-          name: noir_wasm
```

Comment:
noir_wasm only does compilation and no optimizations so this was changed to `nargo_wasm`